admin/controllers/users/views/default/model.php

```diff
@@ -16,9 +16,58 @@
 
 $user = new User();
 
-if ($group_id) {
+/* if ($group_id) {
     $all_users = $user->get_all_users_in_group($group_id);
 }
 else {
     $all_users = $user->get_all_users();
-}
\ No newline at end of file
+} */
+
+// new user search - based on improved content search
+
+$order_by = Input::getvar('order_by','STRING');
+$search = Input::getvar('search','TEXT',null);
+$filters = Input::tuples_to_assoc( Input::getvar('filters','RAW',null) );
+$coretags = Input::getvar('coretags','ARRAYOFINT',[]);
+$groups = Input::getvar('groups','ARRAYOFINT',[]); 
+
+$content_type_filter = null;
+if (sizeof($segments)==3) {
+	$content_type_filter = $segments[2];
+}
```

Comment:
appears this isnt used anywhere? perhaps left over copy and paste? noticed it since content was in the variable name rather than user

Reply:
Think this may have been renamed/used as the group filter instead of the content type, but during development I decided to drop the groups from the main nav and so this wouldn't be used in the main user search. Tidying up code now.